models/src/main/scala/coop/rchain/models/rholang/sorter/ExprSortMatcher.scala

```diff
@@ -213,7 +231,9 @@ private[sorter] object ExprSortMatcher extends Sortable[Expr] {
           connectiveUsedScore = if (list.connectiveUsed) 1 else 0
         } yield
           constructExpr(
-            EListBody(list),
+            EListBody(
```

Comment:
Sorry for not catching those during review ;/

Reply:
Sorry for introducing this bug. Should have known better. OTOH now we have tests for that :P